	For TeX Live we build a static library and not a Dll.

diff -ur teckit-2.5.6.orig/source/Engine.cpp teckit-2.5.6/source/Engine.cpp
--- teckit-2.5.6.orig/source/Engine.cpp	Tue May 31 05:37:08 2016
+++ teckit-2.5.6/source/Engine.cpp	Fri Dec 30 18:18:57 2016
@@ -51,12 +51,6 @@
 #	define NOSERVICE
 #	define NOMCX
 #	include <windows.h>
-
-	BOOL WINAPI
-	DllMain(HINSTANCE /*hInst*/, DWORD /*wDataSeg*/, LPVOID /*lpReserved*/)
-	{
-		return true;
-	}
 #endif
 
 #include "Engine.h"
diff -ur teckit-2.5.6.orig/source/Public-headers/TECkit_Compiler.h teckit-2.5.6/source/Public-headers/TECkit_Compiler.h
--- teckit-2.5.6.orig/source/Public-headers/TECkit_Compiler.h	Tue May 31 05:37:08 2016
+++ teckit-2.5.6/source/Public-headers/TECkit_Compiler.h	Fri Dec 30 18:21:02 2016
@@ -30,6 +30,9 @@
 #ifdef _WIN32
 /* MS compiler has predefined _WIN32, so assume Windows target  */
 #include <windows.h>	// apparently just using windef.h fails on VC++6
+#undef WINAPI
+#define WINAPI
+#define EXPORTED
 #else
 /* not the MS compiler, so try Metrowerks' platform macros */
 #ifndef __APPLE__
diff -ur teckit-2.5.6.orig/source/Public-headers/TECkit_Engine.h teckit-2.5.6/source/Public-headers/TECkit_Engine.h
--- teckit-2.5.6.orig/source/Public-headers/TECkit_Engine.h	Tue May 31 05:37:08 2016
+++ teckit-2.5.6/source/Public-headers/TECkit_Engine.h	Fri Dec 30 18:22:04 2016
@@ -80,6 +80,9 @@
 #ifdef _WIN32
 /* MS compiler has predefined _WIN32, so assume Windows target  */
 #include <windows.h>
+#undef WINAPI
+#define WINAPI
+#define EXPORTED
 #else
 /* not the MS compiler, so try Metrowerks' platform macros */
 #ifndef __APPLE__
